lisa/tools/gcc.py

```diff
@@ -24,5 +24,8 @@ def compile(self, filename: str, output_name: str = "") -> None:
 
     def _install(self) -> bool:
         posix_os: Posix = cast(Posix, self.node.os)
-        posix_os.install_packages("gcc")
+        if isinstance(self.node.os, Debian):
+            posix_os.install_packages("gcc", signed=False)
```

Comment:
have you run the latest code, it used signed = false by default now.

Reply:
Thanks for the comment. I deleted the change of gcc.py.